apps/pie-storybook/stories/contribution/overview.mdx

```diff
@@ -0,0 +1,220 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Contribution/Overview" />
+
+# PIE web components - Contribution guide
+
+This guide walks you through the full contribution process for the PIE web component library, from setup to pull request.
+
+## Table of Contents
+
+1. [Get involved](#1-get-involved)
+2. [Machine setup](#2-machine-setup)
+3. [Clone the repository](#3-clone-the-repository)
+4. [Scaffold a new component](#4-scaffold-a-new-component)
+5. [Develop and test the component](#5-develop-and-test-the-component)
+6. [Component package anatomy](#6-component-package-anatomy)
+7. [Testing standards](#7-testing-standards)
+8. [Create a pull request](#8-create-a-pull-request)
+9. [Add a changeset (if needed)](#9-add-a-changeset-if-needed)
+10. [Trigger a snapshot release](#10-trigger-a-snapshot-release)
+11. [References](#11-references)
+
+## 1. Get involved
+
+Before starting a new component, please contact the Design System team. They will help define requirements and ensure the component is suitable for the shared library.
+
+## 2. Machine setup
+
+Set up your development environment for the PIE monorepo.
+
+### Required tools
+
+- Install a code editor ([VSCode](https://code.visualstudio.com/) recommended)
+- (Optional) Install the [PIE VSCode extension](https://marketplace.visualstudio.com/items?itemName=JamieMaguire.pie-design-system-vscode-prototype)
```

Comment:
Given we have [.vscode/extensions.json](https://github.com/justeattakeaway/pie/blob/main/.vscode/extensions.json), we should probably tell people to install all recommended extensions by navigating to the `Extensions` tab and filtering by recommended.

![Screenshot 2025-06-20 at 11 52 32](https://github.com/user-attachments/assets/7ec5b245-611b-4b5d-9029-565eee41b264)

Weirdly, despite your extension being in this file, and visible in the browser, it doesn't seem to appear in VSCode, even when doing a search for it.

Reply:
good idea!